item.go

```diff
@@ -1 +1,93 @@
 package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/couchbase/gocb"
+)
+
+//Item struct contains properties for a standard item, not all properties are required
+type Item struct {
+	ItemID   string `json:"itemid,omitempty"`
+	ItemName string `json:"itemname,omitempty"`
+	ItemDesc string `json:"itemdesc,omitempty"`
+	Category string `json:"itemcat" `
+	//itempic
+	Price      string `json:"price,omitempty"`
+	DatePosted string `json:"date,omitempty"`
+	SaleStatus string `json:"status,omitempty"`
+	SellerID   string `json:"sellerid,omitempty"`
+}
+
+//Item2 struct
+type Item2 struct {
+	Item Item `json:"itemname"`
+}
+
+//NewItemReq struct for creating new items
+type NewItemReq struct {
+	Itemname string `json:"itemname"`
+	Itemcat  string `json:"itemcat"`
+	Itemdesc string `json:"itemdesc"`
+	Price    string `json:"itemprice"`
+	//item picture coming up
+}
+
+//findItems takes a corkboard object and queries couchbase
+func (corkboard *Corkboard) findItems() ([]Item, error) {
+
+	//META(default).id = Items1
+	//TODO: verify object is of type item
+	query := gocb.NewN1qlQuery(fmt.Sprintf("SELECT name FROM `%s` WHERE itemname = 'bike'", corkboard.Bucket.Name()))
+	log.Println(corkboard.Bucket.Name())
+	rows, err := corkboard.Bucket.ExecuteN1qlQuery(query, []interface{}{})
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	defer rows.Close()
+
+	var item = new(Item)
+	var items []Item
+	for rows.Next(item) {
+		items = append(items, *item)
+	}
+	return items, err
```

Comment:
Shouldn't this return items, nil if there is no error and _, err if there is an error? Doesn't seem to make sense to return both at the same time here.